### DISCUSSION TITLE
Message: Host is not changed.

### DISCUSSION THREAD

**Discussion by shahnazari** [w2c/letsencrypt-esxi#18] [Q&A]
Although I have my FQDN set, vib installation doesn't recognize it and fails with error "Message: Host is not changed." Should there be a server reboot required?

**Comment by Churro** [comment]
No, a reboot is not required.

The message could be shown if the VIB is already installed, effectively meaning the re-installation hasn't changed the host. Can you please check `/var/log/syslog.log`? The [Troubleshooting](https://github.com/w2c/letsencrypt-esxi/wiki/Troubleshooting) wiki could also be insightful.
